spyder/plugins/completion/providers/languageserver/provider.py

```diff
@@ -64,16 +64,11 @@ class LanguageServerProvider(SpyderCompletionProvider):
         ('no_linting', False),
         ('formatting', 'autopep8'),
         ('format_on_save', False),
-        ('pycodestyle', False),
-        ('pycodestyle/filename', ''),
-        ('pycodestyle/exclude', ''),
-        ('pycodestyle/select', ''),
-        ('pycodestyle/ignore', ''),        
         ('flake8/filename', ''),
         ('flake8/exclude', ''),
         ('flake8/extendSelect', ''),
-        ('flake8/extendIgnore', ''),
-        ('pycodestyle/max_line_length', 79),
```

Comment:
Due to this changes in the `CONF_DEFAULTS` shouldn't the `CONF_VERSION` bellow be changed?